_includes/desktop-linux-requirements.md

```diff
@@ -0,0 +1,48 @@
+<!-- This text will be included in the Docker Desktop install prerequisites for each Linux distribution -->
+
+This section contains host configuration requirements for installing and running Docker Desktop.
+
+## KVM virtualisation support
```

Comment:
🇬🇧 As we are standardising on US english 🇺🇸 ; this should be `virtualization` (with a `z`)

```suggestion
## KVM virtualization support
```